src/mlpack/core/util/param_checks_impl.hpp

```diff
@@ -306,6 +306,29 @@ inline void ReportIgnoredParam(util::Params& params,
   }
 }
 
+// Check if the given input data points aren't empty.
```

Comment:
We should document this function more completely, including the assumptions inherent in the function (like that `paramName` is an `arma::mat` type for instance).

Reply:
I changed that to template variable and get values from the different function calls. I will document it.